src/components/Card/DBCard.tsx

```diff
@@ -1,40 +1,82 @@
+'use client';
+
 import { DBCardArrowIcon } from '@/public/index';
+import { useDBCardStore } from '@/stores/DBCardStore';
 import { format } from 'date-fns';
+import Link from 'next/link';
 import { twMerge } from 'tailwind-merge';
 
 type DBCardProps = {
+  id: number;
   title: string;
   date: Date;
   backgroundId: number;
 };
 
-export default function DBCard({ title, date, backgroundId }: DBCardProps) {
-  const backgroundById = (id: number) => {
-    if (id === 1) {
+export default function DBCard({ id, title, date, backgroundId }: DBCardProps) {
+  const { isHoverFirstCard, setIsHoverFirstCard } = useDBCardStore();
+  const conditionOfFirstDBCard = backgroundId === 1 && isHoverFirstCard;
+  const backgroundById = (bgId: number) => {
+    if (bgId === 1) {
       return 'bg-[linear-gradient(360deg,rgba(0,0,0,0.7),rgba(255,255,255,0)),url(/images/DBCardDummyImg1.png)]';
-    } else if (id === 2) {
+    } else if (bgId === 2) {
       return 'bg-[linear-gradient(360deg,rgba(0,0,0,0.7),rgba(255,255,255,0)),url(/images/DBCardDummyImg2.png)]';
     } else {
       return 'bg-[linear-gradient(360deg,rgba(0,0,0,0.7),rgba(255,255,255,0)),url(/images/DBCardDummyImg3.png)]';
     }
   };
+  const onMouseOver2And3 = () => {
+    if (backgroundId !== 1) {
+      setIsHoverFirstCard(false);
+    }
+  };
+  const onMouseOut2And3 = () => {
+    if (backgroundId !== 1) {
+      setIsHoverFirstCard(true);
+    }
+  };
+
   return (
     <div
+      onMouseOver={onMouseOver2And3}
+      onFocus={onMouseOver2And3}
+      onMouseOut={onMouseOut2And3}
+      onBlur={onMouseOut2And3}
```

Comment:
```tsx
const hoverHandlers = {
  onMouseOver: () => handleHover(false),
  onFocus: () => handleHover(false),
  onMouseOut: () => handleHover(true),
  onBlur: () => handleHover(true),
};
```

```suggestion
     {...hoverHandlers}
```
P4: 저도 코드리뷰 남에꺼 훔쳐보고 츄베릅 한건데 이렇게 하면 깔끔하게 관리가 가능할거 같습니다.

Reply:
감사합니다.  훨씬 깔끔해진 것 같네요!